cleanup: enhance info wrt output fields in style guide (#1094)
falcosecurity/falco-website: content/en/docs/rules/style-guide.md

```yaml
user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags
```

For network-related rules include:

```yaml
connection=%fd.name lport=%fd.lport rport=%fd.rport fd_type=%fd.type
```

For file-related rules include:

```yaml
file=%fd.name
```
... and additional specialized fields from the raw args if applicable, such as `newpath=%evt.arg.newpath` for non-file descriptor events like symlinking or renaming.

<!--
TODO: Edit after Falco 0.36.0 release:

For file-related rules include:

```yaml
file=%fd.name
```

Alternatively, you can explore more recent `fs.path.*` variants to simplify the consistent logging of file or directory paths, even for non-file descriptor events like symlinking or renaming. Previously, tapping into the raw args with `newpath=%evt.arg.newpath` was required for such events.
-->

> When writing a rule for container workloads, you should include the fields we automatically fetch from the container runtime (`-k/-K` is *not* required here). All of these fields are incredibly crucial for effective incident response and play a vital role in determining the workload owner:

Reviewer: ```suggestion
When writing a rule for container workloads, you should include the fields we automatically fetch from the container runtime. Falco supports a special placeholder for this purpose: 

```yaml
%container.info
\```

This special placeholder will be automatically replaced with common container-related fields and, when used in conjunction with `-pk` (see the [CLI arguments page](https://falco.org/docs/reference/daemon/cli-arguments/) for more details). It will also add [basic Kubernetes fields](https://falco.org/docs/reference/rules/supported-fields/#field-class-k8s) (this does *not* require the *Kubernetes Metadata Enrichment*, i.e. `-k/-K`,  functionality to be enabled). Using `%container.info` in conjunction with `-pk` is equivalent to:
```

Note I'd to put `\` at line 175 to avoid escaping issue with the GH UI. Please, remove it.

```yaml

> image=%container.image.repository image_tag=%container.image.tag container_name=%container.name namespace=%k8s.ns.name pod_name=%k8s.pod.name

Reviewer: ```suggestion
container_image=%container.image.repository container_image_tag=%container.image.tag container_name=%container.name k8s_ns=%k8s.ns.name k8s_pod_name=%k8s.pod.name
```

```

> 

Reviewer: ```suggestion

All of these fields are incredibly crucial for effective incident response and play a vital role in determining the workload owner.

```

We kindly ask you to add fields related to IDs later in your customization process to keep the upstream Falco output fields to a minimum. This is because there are many ID-related fields, such as `%proc.pid %proc.ppid %proc.vpid %proc.pvpid %proc.sid %proc.vpgid ...`. You can explore the `-p` option for this purpose and add it to all rules' output fields.

Falco also supports outputting the output as a resolved string. Therefore, use a sentence style, first concisely re-iterating the rule's purpose, and then including the output field in parentheses after the `=` character, with its meaning explained before the `=` character.

```yaml
output: >
    Read monitored file via directory traversal (file=%fd.name fileraw=%fd.nameraw 
    gparent=%proc.aname[2] ggparent=%proc.aname[3] gggparent=%proc.aname[4] user=%user.name 
    user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath 
    parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags 

>     image=%container.image.repository image_tag=%container.image.tag 
>     container_name=%container.name namespace=%k8s.ns.name pod_name=%k8s.pod.name)

Reviewer: ```suggestion
    %container.info)
```
